# Adiconis First Alignments
```
pi:ababaian
start: 2016 04 16
complete : 2016 04 28
```

## Objective
* Align the acquired Adiconis K562 data to rRNA


## Methods

### Build HGR Index

```
cd ~/Crown/resources/hgr
bowtie2-build hgr.fa hgr
```
### SRR567450 Alignment 

```
cd ~/Crown/data/adaconis/totalRNA_1

tophat2 --rg-id adiconis_1 --rg-sample k562 ~/Crown/resources/hgr/hgr SRR567450_1.fastq.gz SRR567450_2.fastq.gz
```
Run started: 2016-04-14 16:41:16
Run ended: 2016-04-14 17:56:53

### SRR567451 Alignment 

```
cd ~/Crown/data/adaconis/totalRNA_1

tophat2 --rg-id adiconis_1 --rg-sample k562 ~/Crown/resources/hgr/hgr SRR567451_1.fastq.gz SRR567451_2.fastq.gz
```
Run started: 2016-04-15 09:59:03
Run ended: 2016-04-14 11:04:40

### Redo Alignments: Accidently deleted - 0418

Accidently deleted the above alignments and fastq files when trying to add them to the git repository. Note:  Don't use `git rm -f <file>` to unstage a file '-_-

Regenerating using the same commands and re-aligning. Old files were deleted.

2016-04-18 17:21:55 both runs complete.

## Results

### Flagstats for files


### First Impressions

To take a rough overview of the data IGV was used. I'm having trouble viewing the data, especially over regions of mature rRNA (18S or 28S). Make sure to 'downsample' the reads in View > Preferences > Reads according to your computer. I'm using Max: 1000, per window: 50.

The experiment most certainly worked in the way one would imagine.


In [12]:
# Alignments to rDNA
# K562 TotalRNA 1 file
cd ~/Crown/data/adaconis/totalRNA1
echo "Mapped Reads"
samtools flagstat accepted_hits.bam
echo "Unmapped Reads"
samtools flagstat unmapped.bam

17141165 + 0 in total (QC-passed reads + QC-failed reads)
46794 + 0 secondary
0 + 0 supplimentary
0 + 0 duplicates
17141165 + 0 mapped (100.00%:-nan%)
17094371 + 0 paired in sequencing
8723515 + 0 read1
8370856 + 0 read2
1984966 + 0 properly paired (11.61%:-nan%)
14505026 + 0 with itself and mate mapped
2589345 + 0 singletons (15.15%:-nan%)
0 + 0 with mate mapped to a different chr
0 + 0 with mate mapped to a different chr (mapQ>=5)
6718791 + 0 in total (QC-passed reads + QC-failed reads)
0 + 0 secondary
0 + 0 supplimentary
0 + 0 duplicates
0 + 0 mapped (0.00%:-nan%)
6718791 + 0 paired in sequencing
3183066 + 0 read1
3535725 + 0 read2
0 + 0 properly paired (0.00%:-nan%)
0 + 0 with itself and mate mapped
0 + 0 singletons (0.00%:-nan%)
0 + 0 with mate mapped to a different chr
0 + 0 with mate mapped to a different chr (mapQ>=5)


### Total RNA1 File
Total Reads: 23,859,956
Aligned Reads (rRNA): 17,141,165
Unmapped Reads: 6,718,791

rRNA Reads Mapped: **71.84%**

In [13]:
# Alignments to rDNA
# TotalRNA 2 file
cd ~/Crown/data/adaconis/totalRNA2
echo "Mapped Reads"
samtools flagstat accepted_hits.bam
echo "Unmapped Reads"
samtools flagstat unmapped.bam

Mapped Reads
15722488 + 0 in total (QC-passed reads + QC-failed reads)
156708 + 0 secondary
0 + 0 supplimentary
0 + 0 duplicates
15722488 + 0 mapped (100.00%:-nan%)
15565780 + 0 paired in sequencing
7937614 + 0 read1
7628166 + 0 read2
1943754 + 0 properly paired (12.49%:-nan%)
13116446 + 0 with itself and mate mapped
2449334 + 0 singletons (15.74%:-nan%)
0 + 0 with mate mapped to a different chr
0 + 0 with mate mapped to a different chr (mapQ>=5)
Unmapped Reads
6246124 + 0 in total (QC-passed reads + QC-failed reads)
0 + 0 secondary
0 + 0 supplimentary
0 + 0 duplicates
0 + 0 mapped (0.00%:-nan%)
6246124 + 0 paired in sequencing
2968338 + 0 read1
3277786 + 0 read2
0 + 0 properly paired (0.00%:-nan%)
0 + 0 with itself and mate mapped
0 + 0 singletons (0.00%:-nan%)
0 + 0 with mate mapped to a different chr
0 + 0 with mate mapped to a different chr (mapQ>=5)


### Total RNA2 File
Total Reads: 21,968,612
Aligned Reads (rRNA): 15,722,488
Unmapped Reads: 6,246,124

rRNA Reads Mapped: **71.57%**

## Discussion
#### 2016-12-29
Closing experiment: further analysis in 20160428_1_Variant_Calling.